install.sh

```diff
@@ -0,0 +1,7 @@
+apt-get update -y
```

Comment:
This file should  have a shebang like, like `#!/usr/bin/env bash`, since it is an executable.